site/src/pages/components/menu.explainer.mdx

```diff
@@ -140,21 +143,21 @@ Similarly to customizable select, we expect everything to be wrapped inside `<me
 
 We should support `<fieldset>` and `<legend>` to help group the menu items together.
 
-We should support specific use cases of interactive elements outside the `<menuitem>`. For example, if the user wants to add an `<input type=search>` element, it should not be wrapped inside a `<menuitem>` and should instead be a sibling of `<menuitem>`s.
+We should support specific use cases of interactive elements that can be found within native operating system menus, as siblings to `<menuitem>` elements. For example, if the user wants to add an `<input type=search>` element, it should not be allowed as a child of a `<menuitem>`. Rather, it should be allowed as the first or last child element of a `<menulist>` or `<menubar>` element.
 
-<img src="/images/menubar-search-input.png" alt="menubar examples with search input as a sibling of menuitems" />
+<img src="/images/menubar-search-input.png" alt="examples of text fields as the first child of menulist popups. The first example shows a text field where a percentage can be adjusted, it is followed by a number of menuitems to zoom or adjust an editable canvas. The second example shows a search text field as the first item in a menulist invoked from a Help menuitem of a menubar. Two additional menuitems follow the search text field in this menulist popup." />
 
 
 ## Questions
 
 
-### When to use Select vs Menu
+### When to use Select vs Menulist
 
-Just like a select element, the menu element has a list of choices to pick from. And with customizable select now available, users might be confused on which one to use.
+Just like a select element, the menulist element will represent a series of choices to pick from. And with customizable select now available, users might be confused on which one to use.
 
 The `<select>` element is a form-associated element and contains `<option>` elements. Each of these `<option>` elements must have a server-readable value. As a user chooses an option, specific events are triggered (change, input) and a new value is set.
 
-On the other hand, the proposed `<menulist>` element should be used to display a list of commands/actions. Each `<menuitem>` should be thought of as a command that is activable. That action could be to open a new nested menulist element or to trigger a specific command. It must adhere to [ARIA guidelines for menu and menubar](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/).
+On the other hand, the proposed `<menulist>` element should be used to display a list of commands/actions. Each `<menuitem>` should be thought of as a command that is activable. That action could be to open a new nested menulist element or to trigger a specific command. It is not generally expected for the invoking element's value or visible label to be modified when activating a `<menuitem>`.  It must adhere to ARIA guidelines for [menu and menubar patterns](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/).
```

Comment:
not part of your change, but I don't think “activable” is a word? And it's not very clear what “command that is activable” is. Maybe we can fix at the same time

```suggestion
On the other hand, the proposed `<menulist>` element should be used to display a list of commands/actions. Each `<menuitem>` should be thought of as performing a specific action. That action could be to open a new nested menulist element or to trigger a specific command. It is not generally expected for the invoking element's value or visible label to be modified when activating a `<menuitem>`.  It must adhere to ARIA guidelines for [menu and menubar patterns](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/).
```